source/pdu/main.c

```diff
@@ -59,22 +59,22 @@ GPIOInitConfig_t gpio_config[] = {
     GPIO_INIT_INPUT(FAN_2_NFLT_GPIO_Port, FAN_2_NFLT_Pin, GPIO_INPUT_OPEN_DRAIN),
     GPIO_INIT_ANALOG(FAN_2_CS_GPIO_Port, FAN_2_CS_Pin),
     // Main Module
-    GPIO_INIT_OUTPUT(MAIN_CTRL_GPIO_Port, MAIN_CTRL_Pin, GPIO_OUTPUT_LOW_SPEED),
+    // Disable software control GPIO_INIT_OUTPUT(MAIN_CTRL_GPIO_Port, MAIN_CTRL_Pin, GPIO_OUTPUT_LOW_SPEED),
     GPIO_INIT_INPUT(MAIN_NFLT_GPIO_Port, MAIN_NFLT_Pin, GPIO_INPUT_OPEN_DRAIN),
     GPIO_INIT_ANALOG(MAIN_CS_GPIO_Port, MAIN_CS_Pin),
     // Dashboard
-    GPIO_INIT_OUTPUT(DASH_CTRL_GPIO_Port, DASH_CTRL_Pin, GPIO_OUTPUT_LOW_SPEED),
```

Comment:
On second thought, I know it is dangerous to be able to turn off these control pins, but If you remember FCTT last year and our last PER23 test, it might actually be helpful to keep these controllable by PDU, and have them activated by a DAQ command. The last test, we had to leave the dashboard enclosure open to reset the board because of some weird startup issue that can be mitigtated if we can just reset dashboard here by turning it off and on again. Same with A-Box, where in FCTT a software bug would trigger a fatal fault if you hit the brakes too hard. In both situations, we had no bootloader control so we had to turn the car off completely which wasted time. What if we could just toggle these boards through software? I know it is risky, but it is definitely doable and if these inits are by default open drain and it works fine (all pins on startup are left open drain), then we shouldn't really have a problem, even if PDU is hardfaulting. Just some food for thought.